Gems/Atom/RPI/Code/Include/Atom/RPI.Public/Pass/FullscreenTrianglePass.h

```diff
@@ -28,13 +28,13 @@ namespace AZ
         //! and instead issues a 3-vertex draw and relies on the vertex shader
         //! to generate a fullscreen triangle from the vertex ids.
         class FullscreenTrianglePass
-            : public RenderPass
+            : public RasterPass
```

Comment:
In terms of immediate need: Because SkyBox pass is a FullscreenTrianglePass, and we merge it with the forward pass. And it opens the opportunity  for other FullscreenTrianglePasses to be merged as needed by developers, without having to create a new C++ class.

In terms of API coherency: Any Pass that uses the Raster Pipeline should be a RasterPass regardless where the Draw Items come from. ComputePass subclass RenderPass and they are not mergeable because they don't use the Raster Pipeline.

Reply:
Talked to @antonmic, and he said it is best to leave the FTP as RenderPass because:
```
The raster pass and fullscreen triangle pass diverge in how they deal with things like the pass shader resource group, and having one inherit from the other makes it messy.
```
Solution:
Added a virtual function to RenderPass:  
```cpp
    //! Can instances of this class be merged as subpasses?
    virtual bool CanBecomeSubpass() { return false; }
```
Both RasterPass and FullscreenTrianglePass return **true**.
And all the RenderAttachmentConfiguration related functionality is now consolidated under RenderPass().